tests/import_test.go

```diff
@@ -223,3 +223,47 @@ func startPrometheusPortForward(f *framework.Framework) (string, *exec.Cmd, erro
 
 	return url, cmd, nil
 }
+
+var _ = Describe("Importer Test Suite-Block_device", func() {
+	f := framework.NewFrameworkOrDie(namespacePrefix)
+	var pv *v1.PersistentVolume
+	var err error
+
+	BeforeEach(func() {
+		By(fmt.Sprintf("Creating Block PV"))
+		pv, err = f.CreatePVFromDefinition(utils.NewBlockPVDefinition("local-volume", "1G", nil, "manual"))
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Verify that PV's phase is Available")
+		err = f.WaitTimeoutForPVReady(pv.Name, 60*time.Second)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	AfterEach(func() {
+		err = utils.DeletePV(f.K8sClient, pv)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	It("Should create import pod for block pv", func() {
+		httpEp := fmt.Sprintf("http://%s:%d", utils.FileHostName+"."+utils.FileHostNs, utils.HTTPNoAuthPort)
+		pvcAnn := map[string]string{
+			controller.AnnEndpoint: httpEp + "/tinyCore.iso",
+		}
+
+		By(fmt.Sprintf("Creating PVC with endpoint annotation %q", httpEp+"/tinyCore.iso"))
+		pvc, err := f.CreatePVCFromDefinition(utils.NewBlockPVCDefinition(
```

Comment:
I might be missing something, but this doesn't appear to make a PVC linked to the PV you just created, it creates a PVC using the default storage class but with block volume mode, but the default storage class is not a block volume storage. So the PVC will never bind to the PV you want.

Reply:
I do not know why I passing the storageclass initially. The pvc can be bound to the pv regardless the storage class as long as they match. I removed the 'manual' storage class and using the default. Let me know if I'm missing something...